libs/custom-paper-handler/src/cli/driver_cli.ts

```diff
@@ -51,11 +63,17 @@ async function outputSampleBallotPdfWidth() {
   console.log(`First page of sample ballot is ${page.width} dots`);
 }
 
+async function scan(driver: PaperHandlerDriverInterface): Promise<void> {
+  const dateString = new Date().toISOString();
+  const pathOut = join('/tmp', `ballot-driver-cli-${dateString}.jpg`);
```

Comment:
Similar note as above. Consider `tmpNameSync()`, possibly with options to add `.jpeg` extension.

Reply:
Since the CLI is used only for debugging on the hardware, I think writing to `/tmp` with a human-readable name is the desired behavior here. What do you think?

Reply:
I could buy that. Consider using `os.tmpdir()`?

Reply:
[d00818c](https://github.com/votingworks/vxsuite/pull/3756/commits/d00818cc97ffa81cfbe7ea6ae569d2be1195ee4c)